app/src/main/java/com/example/poste/activities/AccountActivity.java

```diff
@@ -112,14 +112,14 @@ public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response)
                         if (errorMessage != null) {
                             Toast.makeText(AccountActivity.this, errorMessage, Toast.LENGTH_SHORT).show();
                         } else {
-                            Toast.makeText(AccountActivity.this, "Password change failed, unknown error", Toast.LENGTH_SHORT).show();
+                            Toast.makeText(AccountActivity.this, getString(R.string.password_change_failure), Toast.LENGTH_SHORT).show();
                         }
                     }
                 }
 
                 @Override
                 public void onFailure(Call<ResponseBody> call, Throwable t) {
-                    Toast.makeText(AccountActivity.this, "Edit failed, unknown error", Toast.LENGTH_SHORT).show();
+                    Toast.makeText(AccountActivity.this, getString(R.string.edit_failure), Toast.LENGTH_SHORT).show();
```

Comment:
Should this not match the messages from above? The ones above mention passwords, but this one just says "Edit failed"

Reply:
it already does.
![image](https://github.com/Appyo-Poste/Poste/assets/127570704/795eef0a-48ed-4566-baf1-2874bda1c2a2)

Reply:
What I'm saying is, the above failure case says "Password change failed" but this one says "Edit failed". They do not match.